doc/manual/src/expressions/meson.build

```diff
@@ -0,0 +1,42 @@
+doc_manual_src_expressions_dir =  meson.current_source_dir()
+
+# Generate `doc/manual/src/expressions/builtins.md`
+#============================================================================
+
+builtins_dot_json = custom_target(
+    'builtins.json',
+    output: 'builtins.json',
+    command: [
+        nix_bin,
+        '__dump-builtins',
+    ],
+    capture: true,
+    env: dummy_env + ['NIX_PATH=nix/corepkgs=corepkgs'],
+    install: true,
+    install_dir: mandir,
+    build_by_default: true,
+)
+
+builtins_dot_md = custom_target(
+    'builtins.md',
+    input: [
+        nix_bin,
+        builtins_dot_json,
+        join_paths(doc_manual_dir, 'generate-builtins.nix')
+    ] + files(
+        'builtins-prefix.md',
+        'builtins-suffix.md',
+    ),
+    output: 'builtins.md',
+    command: [
+        find_program(
+            'write-builtins.sh',
+            dirs: [join_paths(scripts_dir, 'manual')],
+        ),
+        '@INPUT@',
+    ],
+    capture: true,
+    install: true,
+    install_dir: mandir,
+    build_by_default: true,
+)
```

Comment:
EOL@EOF

Reply:
What does that mean?
Should I remove the newline at the end of the file?

Reply:
github is showing theres isnt a newline at the end of this file :/

Reply:
```shell
xxd -s "$(($(wc -c doc/manual/src/expressions/meson.build | cut -d' ' -f1)-1))" doc/manual/src/expressions/meson.build
00000403: 0a
```

Reply:
> github is showing theres isnt a newline at the end of this file :/

AFAIR, it shows something when there is none, nothing when there is one, and an extra line when there are two.

Reply:
okay, seems to just be a github bug